ui/oopsyraidsy/data/06-ew/raid/p1n.ts

```diff
@@ -0,0 +1,89 @@
+import NetRegexes from '../../../../../resources/netregexes';
+import ZoneId from '../../../../../resources/zone_id';
+import { OopsyData } from '../../../../../types/data';
+import { OopsyTriggerSet } from '../../../../../types/oopsy';
+
+export interface Data extends OopsyData {
+  spell?: { [name: string]: 'cold' | 'hot' };
+}
+
+const triggerSet: OopsyTriggerSet<Data> = {
+  zoneId: ZoneId.AsphodelosTheFourthCircle,
+  damageWarn: {
+    'P1N Gaoler\'s Flail 1': '6DA2', // left/right 180 cleave
+    'P1N Gaoler\'s Flail 2': '6DA3', // left/right 180 cleave
+    'P1N Powerful Light': '65ED', // light explosion during fire/light
+    'P1N Powerful Fire': '65EC', // fire explosion during fire/light
+  },
+  damageFail: {
+    'P1N Painful Flux': '65F2', // standing in cross between hot/cold sections
+  },
+  soloWarn: {
+    'P1N True Holy': '65E7', // Pitiless Flail stack marker
+  },
+  triggers: [
+    {
+      id: 'P1N Cold Spell Track',
+      type: 'GainsEffect',
+      // No need to track this falling off or deleting it.
+      // When the next round comes along, it will reassign and give it to somebody.
+      // TODO: not sure if somebody dies if this falls off and so maybe we should track losing it?
```

Comment:
I *think* you're correct, we don't need to track losing the debuffs. Unfortunately I don't have anything in my logs for "people being dead during hot/cold".